lib/eventhandlers.js

```diff
@@ -91,7 +91,7 @@ var EventHandlers = {
             }
         }
 
-        inputmask.ignorable = opts.ignorables.includes(k);
+        inputmask.ignorable = k ? opts.ignorables.includes(k) : false;
```

Comment:
I think you can change this to:
```js
inputmask.ignorable = k && opts.ignorables.includes(k);
```

Reply:
Just keep  inputmask.ignorable = opts.ignorables.includes(k); and remove 0 from the ignorables in the defaults.js